react-app/src/app/components/Board/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import Square from '../Square';
+
+import './styles.css';
+
+class Board extends Component {
+  static propTypes = {
+    squares: PropTypes.Array,
+    onClick: PropTypes.func
+  };
```

Comment:
This should be:
```
Board.propTypes = {
    squares: PropTypes.Array,
    onClick: PropTypes.func
}
```
Outside the class

Reply:
Also, Array is pretty generic. You should use `arrayOf(PropTypes.String)` For example